src/components/intents/IntentHandler.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Children, Component } from "react";
```

Comment:
This whole file comes from cozy-store, and while the internals are what they are, the API is pretty sweet.